packages/apps/composer-app/vite.config.ts

```diff
@@ -88,6 +91,19 @@ export default defineConfig({
         resolve(__dirname, '../plugins/*/src/**/*.{js,ts,jsx,tsx}'),
       ],
     }),
+    IconsPlugin({
+      symbolPattern:
+        'ph--([a-z]+[a-z-]*)--(bold|duotone|fill|light|regular|thin)',
+      assetPath: (name, variant) =>
+        `${phosphorIconsCore}/${variant}/${name}${
+          variant === 'regular' ? '' : `-${variant}`
+        }.svg`,
+      spritePath: resolve(__dirname, 'public/icons.svg'),
+      contentPaths: [
+        `${resolve(__dirname, '../../..')}/{packages,tools}/**/dist/**/*.{mjs,html}`,
```

Comment:
tools needs to be included here?

Reply:
I couldn’t be sure, so I allowed it just in case. Can be removed if needed, it’s not actually looking in there just running `picomatch` to determine whether to check if a file might have symbols in it or not.